docs/content/en/docs/tasks/cluster/cluster-upgrades.md

```diff
@@ -46,6 +47,30 @@ version.
 Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
 Anywhere components mentioned above to their latest available versions. All upgrade changes are backwards compatible.
 
+### Check upgrade components
+Before you perform an upgrade, check the current and new versions of components that are ready to upgrade by typing:
+
+```bash
+eksctl anywhere upgrade plan cluster -f cluster.yaml
+```
+The output should appear similar to the following:
+
+```
+Worker node group name not specified. Defaulting name to md-0.
+Warning: The recommended number fo control plane nodes is 3 or 5
```

Comment:
Is "fo" a bug in the code or just in our example output?

Reply:
That was me. Fixed.